coalib/core/Bear.py

```diff
@@ -85,7 +85,7 @@ class Bear:
     ... 'Smell', 'Unused Code', 'Redundancy', 'Variable Misuse', 'Spelling',
     ... 'Memory Leak', 'Documentation', 'Duplication', 'Commented Code',
     ... 'Grammar', 'Missing Import', 'Unreachable Code', 'Undefined Element',
-    ... 'Code Simplification'}
+    ... 'Code Simplification', 'License'}
```

Comment:
Changes looks good. Just update the commit message, and it should be good to merge.